tables/rx_potency/edara22-pot.csv

```diff
@@ -0,0 +1,4 @@
+ref_name,rx_name,iso_name,section,assay_name,potency_type,potency,cumulative_count,potency_upper_limit,potency_lower_limit,potency_unit,date_added
+Edara22,BBV152,S:614G,Figure 1,Virus isolate,NT50,706,13,10000,10,NULL,2022-04-12
```

Comment:
If the study uses authentic virus (virus isolate here) for testing NT50s, the control should be a full genome isolate such as `B.1 full genome`.

Reply:
Ah, is S:614G not a full genome isolate? Or do you mean use isolates that say `full genome` in its name?

Reply:
> Ah, is S:614G not a full genome isolate? Or do you mean use isolates that say `full genome` in its name?

For live virus test saying used D614G as the control, most time it's actually a B.1 virus. Using S:614G or B.1 full genome doesn't make a difference when we displaying the results so we don't have to go back and change what's in our database. But it is preferred to use B.1 full genome instead of S:614G when describing live virus neutralization test.